### PR TITLE
Feature/faq

### PR DESCRIPTION
**Pull request by aayusha-shrestha**
#### What is the purpose of your change?
-Developed FAQ page

#### What areas of the application does this change impact? (changes API, folder structure, etc)
-FAQ section

#### What features/fixes are deferred/not handled in this PR?
-Responsive design

## Checklist
- [ ] Adds unit tests
- [ ] Has a work item linked to the PR

### CODE REVIEW

File: src/pages/faq/Faq.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import { faqs } from "./faqs";
+
+const Faq = () => {
+  const [clicked, setClicked] = useState(false);
+
+  const toggle = (id) => {
+    if (clicked === id) {
+      return setClicked(null);
+    }
+    setClicked(id);
```
Comment (michachan) [inline src/pages/faq/Faq.js:16]: nit: alternatively, setClicked(() => clicked === id ? null : id)

File: src/pages/faq/Faq.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    icon={clicked === faq.id ? faChevronUp : faChevronDown}
+                    style={{ color: "#fae9d3" }}
+                    className="chevron"
+                  />
+                </button>
+              </div>
+              <div className="answers">
+                <p>
+                  {clicked === faq.id ? (
+                    <div dangerouslySetInnerHTML={{ __html: faq.body }} />
```
Comment (michachan) [inline src/pages/faq/Faq.js]: why use dangerouslysetinnerhtml?

**Review by michachan — CHANGES_REQUESTED** [review]
Please explain reason why dangerouslysetinnerhtml is needed

**Comment by aayusha-shrestha** [comment]
Thank you. Edited.

**Comment by aayusha-shrestha** [comment]
I apologize for my delayed response. A link needs to be attached to one of
the FAQs. I could not find any alternative to place the anchor tag in the
array.

File: src/pages/faq/Faq.js (hunk 1 above)
Comment (aayusha-shrestha) [reply src/pages/faq/Faq.js:16]: Re-render error occured

**Review by michachan — APPROVED** [review]

**Pull request merged**